src/spec.cr

```diff
@@ -100,6 +109,14 @@ OptionParser.parse do |opts|
       exit 1
     end
   end
+  opts.on("--order MODE", "run examples in random order by passing MODE as 'random' or to a specific seed by passing MODE as the seed value") do |mode|
+    if mode =~ /\A(?:default|random|\d{1,5})\Z/
+      Spec.order = mode
+    else
+      STDERR.puts "order must be either 'default', 'random', or a seed value"
+      exit 1
+    end
```

Comment:
Please use `abort`

Reply:
I can do that, but it's not like that elsewhere in this file (not sure what the general approach is for things like that - stay consistent, or move forward)

Reply:
Done...I can revert back if consistency is preferable